draft-ietf-quic-recovery.md

```diff
@@ -1269,19 +1269,42 @@ timers wake up late. Timers set in the past fire immediately.
 Pseudocode for SetLossDetectionTimer follows:
 
 ~~~
-GetEarliestTimeAndSpace(times):
-  time = times[Initial]
+GetLossTimeAndSpace():
+  time = loss_time[Initial]
   space = Initial
   for pn_space in [ Handshake, ApplicationData ]:
-    if (times[pn_space] != 0 &&
-        (time == 0 || times[pn_space] < time) &&
-        # Skip ApplicationData until handshake completion.
-        (pn_space != ApplicationData ||
-          IsHandshakeComplete()):
-      time = times[pn_space];
+    if (time == 0 || loss_time[pn_space] < time):
+      time = loss_time[pn_space];
       space = pn_space
   return time, space
 
+GetPtoTimeAndSpace():
+  duration = (smoothed_rtt + max(4 * rttvar, kGranularity))
+      * (2 ^ pto_count)
+  # Arm PTO from now when there are no inflight packets
+  if (no in-flight packets):
```

Comment:
Assuming this is for the amplification protection, something's missing here. Perhaps this should be 
`if (is_client && no packets in flight && !PeerCompletedAddressValidation())`.
Otherwise, you never get into the case below on line 1293.

Reply:
never mind, I thought the code below to do the early return when no packets are in flight after address validation was removed. It's still there, so all good.